help-beta/tsconfig.json

```diff
@@ -26,5 +26,7 @@
 
         /* Projects */
         "composite": true,
+
+        "types": ["unplugin-icons/types/astro"],
```

Comment:
I'd use the trailing commas style to have this on a line by itself to reduce future diffs when adding other similar things given this is a list that's likely not going to stay at one element forever, but it's not worth changing -- just the next person adding one will do that.